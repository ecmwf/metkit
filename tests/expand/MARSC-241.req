retrieve,
    DOMAIN=G,LEVTYPE=ML,LEVELIST=1,DATE=20041015,TIME=1200,STEP=0,PARAM=73.210/74.210,CLASS=RD,TYPE=FC,STREAM=OPER,EXPVER=f36f,
    target=reference.data