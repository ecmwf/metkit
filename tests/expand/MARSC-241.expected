RETRIEVE,
    CLASS      = RD,
    TYPE       = FC,
    STREAM     = OPER,
    EXPVER     = f36f,
    REPRES     = SH,
    LEVTYPE    = ML,
    LEVELIST   = 1,
    PARAM      = 210073/210074,
    DATE       = 20041015,
    TIME       = 1200,
    STEP       = 0,
    DOMAIN     = G,
    TARGET     = reference.data